claripy/backends/backend_z3.py

```diff
@@ -767,7 +757,7 @@ def _generic_model(self, z3_model):
         for m_f in z3_model:
             n = _z3_decl_name_str(m_f.ctx.ctx, m_f.ast).decode()
             m = m_f()
-            me = z3_model.eval(m)
+            me = z3_model.eval(m, model_completion=True)
```

Comment:
@zardus Fish told me to ask you to take a look at this line when I pointed out that I thought it should have this parameter.